tradeexecutor/analysis/trade_analyser.py

```diff
@@ -357,6 +358,7 @@ class TradeSummary:
         encoder=json_encode_timedelta,
         decoder=json_decode_timedelta,
     ))
+    is_time_bucket: bool = False
```

Comment:
There is no point to have boolean variable "is_time_bucket" because it does not tell about the time bucket used.

The correct way to resolve this is to have a member `time_bucket: Optional[TimeBucket] = None`